packages/server/src/time-control/timeout.ts

```diff
@@ -104,22 +104,25 @@ export class TimeoutService {
   ): void {
     const timeoutResolver = async () => {
       const game = await getGame(gameId);
-
-      const timeoutMove: MovesType = { [playerNr]: "timeout" };
-      game.moves.push(timeoutMove);
       let timeControl = game.time_control;
 
       // this next part is somewhat duplicated from the playMove function
```

Comment:
I think this is code is looking a lot closer to `playMove()` after this PR.  Is there any reason not to just call `playMove()` from here?

Reply:
I think that would be possible, only I need to circumvent some of the validation.

For example there is a check whether the seat (that is playing a move) is occupied. So when this seat times out, we automatically play a timeout move for it, and it should go through even if the seat is not occupied.

I could add a boolean input to playMove() that disables this validation check (possibly among others).

Reply:
I've taken another look, and have a new proposal on how this could work. But I'd like to make a new PR for that change.